src/plugins/auto/multi_schedule.cpp

```diff
@@ -254,23 +254,28 @@ MultiSchedule::~MultiSchedule() {
                     LOG_INFO_TAG("CPU_HELP:fps:%lf", count * 1000 / durtation.count());
                 }
             } else {
-                LOG_INFO_TAG("%s:infer:%ld", _workerRequest.first.c_str(), count);
-                auto n = reqAllStartTimes.size();
                 Time time;
+                // skip the first infer, Calculate fps using the time between the last infer and the second infer
+                // keep count as real count, don't count-- when skipping the first infer
+                if (!reqAllStartTimes.empty()) {
+                    reqAllStartTimes.pop_front();
+                }
                 while (!reqAllStartTimes.empty()) {
                     time = reqAllStartTimes.front();
                     if (time < _cpuHelpReleaseTime) {
                         reqAllStartTimes.pop_front();
-                        n--;
+                        count--;
                     } else {
                         break;
                     }
                 }
-                if (n >= 1) {
-                    std::chrono::duration<double, std::milli> durtation =
-                        reqAllEndTimes.back() - time;
-                    LOG_INFO_TAG("%s:fps:%lf", _workerRequest.first.c_str(),
-                        n * 1000 / durtation.count());
+                LOG_INFO_TAG("%s:infer:%ld", _workerRequest.first.c_str(), count);
+                if (count >= 1) {
+                    std::chrono::duration<double, std::milli> durtation = reqAllEndTimes.back() - time;
```

Comment:
count has been -- to discard the inference happened before cpu help release, it does not reflect the real count, is this your expectation?